docs/pages/kubernetes-access/controls.mdx

```diff
@@ -487,19 +487,75 @@ spec:
       - kind: pod
         namespace: "production"
         name: "webapp"
+        verbs: ['*']
     # ...
 ```
 
-The value of this field is a list of mappings, where each mapping has three
+The value of this field is a list of mappings, where each mapping has four
 fields:
 
-- `kind`: The kind of resource to enable access to. Currently, Teleport only
-  supports the value `pod`.
+- `kind`: The kind of resource to enable access to. Currently, Teleport supports
+the following kinds:
+  - `*`: Allow access to all resources.
```

Comment:
I think we could change this to a table. This way, we could reduce repetition of "Controls access to" and "the `name` field...corresponds to...".

Reply:
236fbd2b919631466e288b1f3a408e18dce5b000